lisa/tests/base.py

```diff
@@ -638,7 +638,8 @@ def wrapper(self, *args,
             )
 
             #pylint: disable=no-member
```

Comment:
That warning was for accessing `wrapper.__doc__` which is now one line below. I guess you could remove it, at the time I added it feel to make it feel like we're progressing towards a pylint-checkable codebase...

Reply:
right, if we want to aim at that, we should add a travis check to make sure that new patches don't introduce pylint warnings, and we can fix the existing warnings a bit at a time during these 3 minutes before lunch break when it feels nothing big should be started :-)